course/assignments/R-intro-lectures-assignments.md

```diff
@@ -0,0 +1,60 @@
+# Week 1 (4 marks) Intro to programming
+1. Why is it beneficial to use a programming language rather than a spreadsheet software for data analyses?
+2. Get setup with Rstudio at home (or a lab computer after hours). Install R, Rstudio, and the necessary packages (tidyverse, rmarkdown). Use `install.packages(<package_name>)`. Load all the libraries you installed into your environment with `library(<package_name>)` and send us the output of running `sessionInfo()`.
+3. What is Rmarkdown and why are we using it in this class? How do you execute a chunk of code in an Rmarkdown document?
+4. Is there anything in particular you are excited about learning about theoretical ecological population models and about using R for data analyses and modelling?
+
+# Week 2 (8 marks) Rmarkdown, assignment, functions, vectors, data frames
```

Comment:
Yes, this is a good idea! I am making a `.Rmd` with all the questions and answers. Then we can include all the structure and even some of the answers if we think they students might need a few extra hints on the tougher questions.